guides/v2.1/cloud/env/environments-start.md

```diff
@@ -12,15 +12,16 @@ functional_areas:
   - Cloud
 ---
 
-When managing with your environment, you will tend to use the Magento CLI and SSH into the system. You should have the Magento CLI installed and SSH keys set up. For detailed information on the environment architecture, see [Starter]({{ page.baseurl }}/cloud/basic-information/starter-architecture.html) or [Pro]({{ page.baseurl }}/cloud/architecture/pro-architecture.html) architecture information.
+After you install the Magento CLI and set up SSH keys for remote access to your Cloud infrastructure, you can use Magento CLI commands to manage the environments for your {{site.data.var.ece}} projects. For information about the environment architecture, see [Starter architecture]({{ page.baseurl }}/cloud/basic-information/starter-architecture.html) or [Pro architecture]({{ page.baseurl }}/cloud/basic-information/pro-architecture.html).
```

Comment:
We don't need "For information about..." because it is fluff.
"See ..." is enough and used throughout.

Also, Magento CLI is interesting. Is it a Magento CLI or a Cloud CLI? I know this isn't your doing, but if it is the cloud CLI, then we should replace with:
`magento-cloud` CLI

That may be scope-creep here, but just noting it.

Reply:
In this case, I intentionally left "for information about the environment architecture" because the previous sentence talks about environments ... and the Architecture topic title does not immediately convey the connection between architecture and environments.

In general, I think we need to clean up the way we use "environment", but I'm not sure what that looks like overall.  This topic needs further work/clarification, but I wanted to get the IP address table fix pushed. 

I replaced the Magento CLI references with Magento Cloud CLI -- easy fix, and it was not correct as it was.